user-docs/latest/content-manager/configuring-view-of-content-type.md

```diff
@@ -1,6 +1,10 @@
-# Configuring the view of a content type
+# Configuring the views of a content type
```

Comment:
Just a comment, wondering if we can decrease the line length to prevent word wrapping (not just this title entry but there are others too)

EX: 
![image](https://user-images.githubusercontent.com/8593673/105515784-58385700-5c92-11eb-8ad5-374cd08e02ff.png)

Reply:
Yes, I have noticed and it annoys me too. I have tried to change the titles but most of the time, I don't manage the make the titles shorter while keeping them all homogeneous and without losing important words. I will think about what I can do, but maybe it will only be something we can change when we will modify the global design of the documentation (much later).